release-doc-in-process/all_vf.TTL

```diff
@@ -346,9 +366,74 @@ vf:fulfills     a owl:ObjectProperty ;
 #        rdfs:range      xsd:boolean ;
 #        rdfs:comment    "The commitment or process is complete or not." .
 
-# isSubstitutable "Defines if any resources of that type can be freely substituted for any other resource of that type when used, consumed, traded, etc."
+vf:isSubstitutable a    owl:DatatypeProperty ;
+        rdfs:label      "is substitutable" ;
+        rdfs:domain     vf:ResourceType ;
+        rdfs:range      xsd:boolean ;
+        rdfs:comment    "Defines if any resource of that type can be freely substituted for any other resource of that type when used, consumed, traded, etc." .
+
+#vf:unit a               owl:ObjectProperty ;
+#        rdfs:label      "unit" ;
+#        rdfs:domain     vf:ResourceType ;
+#        rdfs:range      qudt:unit ;
+#        rdfs:comment    "The unit expected for this resource type as a default." .
+
+vf:definedQuantity a    owl:ObjectProperty ;
+        rdfs:label      "defined quantity" ;
+        rdfs:domain     vf:RecipeFlow ;
+        rdfs:range      qudt:QuantityValue ;
+        rdfs:comment    "The amount and unit defined for this recipe flow." .
+
+# TODO: need to define this, something like currency, inventory, work, service; may be other aspects too.
+#vf:resourceCategory a    owl:ObjectProperty ;
+#        rdfs:label      "category" ;
+#        rdfs:domain     vf:ResourceType ;
+#        rdfs:comment    "Defines the category of behavior expected for the type of resource." .
+
+vf:developedBy a        owl:ObjectProperty ;
+        rdfs:label      "developed by" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      foaf:Agent ;
+        rdfs:comment    "The agent who developed this recipe." .
+
+vf:variantOf a           owl:ObjectProperty ;
+        rdfs:label      "variant of" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      vf:Recipe ;
+        rdfs:comment    "This recipe is a variation of another recipe." .
+
+vf:successorOf a        owl:ObjectProperty ;
+        rdfs:label      "successor of" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      vf:Recipe ;
+        rdfs:comment    "This recipe is an updated version of another recipe." .
+
+vf:actsOn a             owl:ObjectProperty ;
+        rdfs:label      "acts on" ;
+        rdfs:domain     vf:RecipeFlow ;
+        rdfs:range      vf:ResourceType ;
+        rdfs:comment    "Defines the type of resource this recipe flow acts on." .
+
+vf:hasDuration a        owl:ObjectProperty ;
```

Comment:
I think sometimes duration could depend on quantities and sometimes not. To assemble 10 bikes it will take around 10x time than 1 bike. On the other hand baking 10 loafs of bread doesn't have to take 2x time of baking 5 loafs.

Reply:
Good point, talked to Bob a while about the complexity of capacity planning, which he has some experience in.  I'm thinking adding a multiplier in RecipeProcess, seems useful.  We might think of other things with some research; or can just leave it as extensible.  For example equipment has a capacity, so the bread multiplier might depend on that.  But the capacity of the oven is different for bread and for cookies.